### PR TITLE
Add sample code to create AML hybrid model.

### PR DESCRIPTION
**Pull request by RandySheriffH**
Implement scripts for creating hybrids:

1. If - customer could switch between edge and remove model with a boolean input;
2. And - customer could infer both in a single run;
3. IfThen - custom could infer edge model first, judge its quality, and go remove if not good enough.

### CODE REVIEW

**Comment by natke** [comment]
Can we use actual models for this sample, rather than toy ones?

**Comment by RandySheriffH** [comment]
> Can we use actual models for this sample, rather than toy ones?
We might - let's see if could use the whisper model from build to showcase this.

**Comment by pranavsharma** [comment]
Why is this in the azureml folder? I suggest we create a new folder called AzureEP and put this file inside it with a README.md.

File: python/azureml/hybrid.py [hunk 1]
```diff
@@ -0,0 +1,316 @@
+# ------------------------------------------------------------------------------------------------------
+# The file implemented three ways to merge a local model and a azure-proxy model for hybrid inferencing:
```
Comment (pranavsharma) [inline python/azureml/hybrid.py:2]: Someone new to this file is going to ask what is a azure-proxy model.

File: python/azureml/hybrid.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
+# ------------------------------------------------------------------------------------------------------
```
Comment (pranavsharma) [inline python/azureml/hybrid.py:1]: 1. Needs license and copyright header.
2. Rename file to create_hybrid_model_using_azure_ep.py.

File: python/azureml/hybrid.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+# The file implemented three ways to merge a local model and a azure-proxy model for hybrid inferencing:
+# 1. Run either one.
```
Comment (pranavsharma) [inline python/azureml/hybrid.py:3]: It'll really help if we can make some ASCII drawings showing what configurations we are building between the edge and proxy models.

**Comment by RandySheriffH** [comment]
> 1. Why is this in the azureml folder? I suggest we create a new folder called AzureEP and put this file inside it with a README.md.
> 2. I assume we're going to add a link to these samples from the https://onnxruntime.ai/docs/execution-providers/Azure-ExecutionProvider.html page?

Yes， there will be link from ort to this page.

File: python/azureml/hybrid.py (hunk 3 above)
Comment (RandySheriffH) [reply python/azureml/hybrid.py:3]: Will move this part to README.md

File: python/azureml/hybrid.py (hunk 2 above)
Comment (RandySheriffH) [reply python/azureml/hybrid.py:1]: How about create_hybrid.py?
The sample is only about onnx graph editing, we will have another doc from ort on AzureEP.

File: python/azureml/hybrid.py (hunk 1 above)
Comment (RandySheriffH) [reply python/azureml/hybrid.py:2]: Explaining this in README.md

File: python/AzureEP/create_hybrid.py [hunk 4]
```diff
@@ -0,0 +1,372 @@
+# -------------------------------------------------------------------------
```
Comment (pranavsharma) [inline python/AzureEP/create_hybrid.py:1]: create_hybrid_model.py?

File: python/AzureEP/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Since onnxruntime 1.16, AzureEP supports the running of a "hybrid" model combining two compoments:
+
+- An edge model thats run locally.
```
[the remark this answers is not in the document]
Comment (RandySheriffH) [reply python/AzureEP/README.md]: Corrected.

File: python/AzureEP/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+- A proxy model talks to a remote endpoint hosted remotely.
```
[the remark this answers is not in the document]
Comment (RandySheriffH) [reply python/AzureEP/README.md]: Done.

File: python/AzureEP/create_hybrid.py (hunk 4 above)
Comment (RandySheriffH) [reply python/AzureEP/create_hybrid.py]: Done.

File: python/AzureEP/README.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+- An edge model that runs locally.
+- A proxy model talks to a remote endpoint hosted hosting a model as a service.
+
+The file implemented three ways to create such a "hybrid" model:
+
+- Run either one.
+- Run both.
+- Run the first model, then the second if need to.
+
+Also, there are two exampels for usage:
```
Comment (jeffmend) [inline python/AzureEP/README.md:12]: examples

File: python/AzureEP/README.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+Since onnxruntime 1.16, AzureEP supports the running of a "hybrid" model combining two compoments:
+
+- An edge model that runs locally.
+- A proxy model talks to a remote endpoint hosted hosting a model as a service.
+
+The file implemented three ways to create such a "hybrid" model:
```
Comment (jeffmend) [inline python/AzureEP/README.md:6]: The file can be implemented in 3 ways

File: python/AzureEP/README.md (hunk 7 above)
Comment (RandySheriffH) [reply python/AzureEP/README.md]: Corrected.

File: python/AzureEP/README.md (hunk 8 above)
Comment (RandySheriffH) [reply python/AzureEP/README.md]: How about - "There are three ways to create such ..."?

**Review by pranavsharma — APPROVED** [review]

**Pull request merged**